bundle/config/resources/secret_scope.go

```diff
@@ -0,0 +1,86 @@
+package resources
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/marshal"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+type SecretScopePermissionLevel string
+
+// SecretScopePermission holds the permission level setting for a single principal.
+// Multiple of these can be defined on any secret scope.
+// Secret scopes permissions are mapped to Secret ACLs
+type SecretScopePermission struct {
```

Comment:
Is this the right design decision? Normally we try to stick with the way API represents stuff. I think we should consider calling this field `acls` and using the API reference which is:
```
permission
principal
scope
```

Implcitly translating permissions to this representation will be more confusing IMO.

ref: https://docs.databricks.com/api/workspace/secrets/putacl

Reply:
as discussed, naming this field `permissions` aligns nicely with other resources types that we support in DABs